src/Util/File.cc

```diff
@@ -10,6 +10,7 @@
 #include <fstream>
 #include <regex>
 
+#include "Casacore.h"
```

Comment:
If this include necessary?

Reply:
Asking because I am looking at a file list bug and might include File.h in Casacore.h.